templates/components/common/wishlist-add.html

```diff
@@ -1,7 +1,29 @@
-<form action="{{add_to_wishlist_url}}" class="js-wishlist-add c-wishlist" method="post">
-    <button class="c-wishlist__button" type="submit">
-        <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none">
-            <path d="M12.62 20.81C12.28 20.93 11.72 20.93 11.38 20.81C8.48 19.82 2 15.69 2 8.68998C2 5.59998 4.49 3.09998 7.56 3.09998C9.38 3.09998 10.99 3.97998 12 5.33998C13.01 3.97998 14.63 3.09998 16.44 3.09998C19.51 3.09998 22 5.59998 22 8.68998C22 15.69 15.52 19.82 12.62 20.81Z" stroke="white" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/>
-        </svg>
-    </button>
-</form>
+{{#if customer '!==' null}}
+    <form 
+        action="{{add_to_wishlist_url}}" 
+        class="js-wishlist-add c-wishlist" 
+        method="post"
+    >
+        <button class="c-wishlist__button" type="submit">
+            <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none">
```

Comment:
Why not  to use`components/icon` ? 🤔

Reply:
This was implemented before the components/icon change and just forgot to replace it. Will do so now